test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -0,0 +1,320 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'base_test'
+
+# Unit tests for Google Cloud Logging plugin
+class GoogleCloudOutputGRPCTest < GoogleCloudPluginBaseTest
+  def test_configure_service_account_application_default
+    verify_configure_service_account_application_default(
+      method(:create_grpc_driver))
```

Comment:
This has been refactored. The code looks cleaner now. Wish I was familiar with the "module" usage. Because I was attempting to do this by using inheritance, which did not work well for the testing framework only recognized tests directly defined in the subclass.